blob/gcsblob/example_test.go

```diff
@@ -18,12 +18,13 @@ import (
 	"context"
 	"log"
 
+	"cloud.google.com/go/storage"
 	"gocloud.dev/blob"
 	"gocloud.dev/blob/gcsblob"
 	"gocloud.dev/gcp"
 )
 
-func Example() {
+func Example_read() {
```

Comment:
Where does this put the example? It was supposed to be a package-level example before.

Reply:
I believe it still puts it at the package level and just names it more explictly as "Example (Read)".

If you look at https://godoc.org/gocloud.dev/blob/gcsblob, at the `As` function, this example comes immediately after it, which is confusing because it's not an example of `As`. Now that we have an actual `As` example, it may be not as important though.

Reply:
OK